addons/account/models/account_move.py

```diff
@@ -1927,7 +1927,9 @@ def copy(self, default=None):
             default['date'] = self.company_id._get_user_fiscal_lock_date() + timedelta(days=1)
         if self.move_type == 'entry':
             default['partner_id'] = False
-        return super(AccountMove, self).copy(default)
+        res = super(AccountMove, self).copy(default)
+        res._recompute_payment_terms_lines()
+        return res
```

Comment:
```
        invoices = super().copy(default)
        invoices._recompute_payment_terms_lines()
        return invoices
```
?

Also, plz add a test

Reply:
@smetl thanks for the review!
Could you check the update, please ?